Mobile Buy SDK/Mobile Buy SDK Tests/BUYClientTest.m

```diff
@@ -327,14 +327,18 @@ - (void)testProductsInCollectionWithSortOrderTitleDescending
 
 - (void)testCustomerCreationURL
 {
-	BUYAccountCredentialItem *firstName = [BUYAccountCredentialItem itemWithKey:@"first_name" value:@"michael"];
-	BUYAccountCredentialItem *lastName = [BUYAccountCredentialItem itemWithKey:@"last_name" value:@"scott"];
-	BUYAccountCredentialItem *email = [BUYAccountCredentialItem itemWithKey:@"email" value:@"fake@example.com"];
-	BUYAccountCredentialItem *password = [BUYAccountCredentialItem itemWithKey:@"password" value:@"password"];
-	BUYAccountCredentialItem *passwordConfirmation = [BUYAccountCredentialItem itemWithKey:@"password_confirmation" value:@"password"];
-	BUYAccountCredentials *credentials = [BUYAccountCredentials credentialsWithItems:@[firstName, lastName, email, password, passwordConfirmation]];
-	
-	NSURLSessionDataTask *task = [self.client createCustomerWithCredentials:credentials callback:nil];
+	NSArray *items = @[
+					   [BUYAccountCredentialItem itemWithFirstName:@"michael"],
+					   [BUYAccountCredentialItem itemWithLastName:@"scott"],
+					   [BUYAccountCredentialItem itemWithEmail:@"fake@example.com"],
+					   [BUYAccountCredentialItem itemWithPassword:@"password"],
+					   [BUYAccountCredentialItem itemWithPasswordConfirmation:@"password"],
+					   ];
+	BUYAccountCredentials *credentials = [BUYAccountCredentials credentialsWithItems:items];
+	
+	NSURLSessionDataTask *task = [self.client createCustomerWithCredentials:credentials callback:^(BUYCustomer *customer, NSString *token, NSError *error) {
+		
```

Comment:
These blocks are empty only to satisfy the `nonnull` constraint for the callback parameter. Prior to the introduction of that constraint we just passed `nil`.

We can't setup an expectation here because the callbacks aren't functional. They never get called because we're using a subclass of `BUYClient` that never resumes the network call task. I'm open to other suggestions for getting rid of the empty blocks but for now, this is the best I could come up with.